docs/config/keys.md

```diff
@@ -20,53 +38,142 @@ return {
 }
 ```
 
-The `action` value can be one of the [available key
-assignments](lua/keyassignment/index.md).  Every action has an example that shows
-how to use it.
+You can also turn off all default keybindings entirely via:
+
+```lua
+return {
+  disable_default_key_bindings = true,
+}
+```
+
+## Creating New Assignment
+
+Key assigments can be defined in your `wezterm.lua` config by adding key config structs to the `keys` table e.g.
+
+```lua
+local wezterm = require 'wezterm'
+
+-- Start of main config table
+return {
+  -- Start of keys table
+  keys = {
+    -- Key config struct 1
+    { key = '0', mods = 'SUPER', action = wezterm.action.ResetFontSize },
+    -- Key config struct 2
+    {
+      key = 'l',
+      mods = 'ALT',
+      action = wezterm.action.ShowLauncher
+    },
+  },
+}
+```
+## Key Config Struct
+
+A key config struct consists of 3 fields:
+
+**Fields:**
+
+- key (required)
+- mods (optional)
+- action (required)
+
+example of possible field assignments: 
+
+| `key = `  |   `mods =`    |           `action =`            |
+| :-------: | :-----------: | :-----------------------------: |
+|    'P'    |    'SUPER'    |   wezterm.action.SpawnWindow    |
+| "Escape"  | 'CTRL\|SHIFT' | wezterm.action.ShowDebugOverlay |
+| "UpArrow" |               |       wezterm.action.Nop        |
+
+
+Combining the 3 fields togeather, we can construct a key assignment structs in our config e.g.
+
+```lua
```

Comment:
this seems to fail at ci/subst-release-info.py with 
```
Error {
    "documentation_url":"https://docs.github.com/rest",
    "message":"Bad credentials"
}
```

Work around: commenting this step from build-docs.sh

additionally, `mdbook build docs` fails to generate because it's expecting to find the chapters:

install/freebsd.md
install/linux.md
install/macos.md
install/source.md
install/windows.md

https://github.com/wez/wezterm/blob/691ec187ba29fcae45ddf4e4f88fd02c49988c86/ci/generate-docs.py#L310

but the files are named:

install/freebsd.markdown
install/linux.markdown
install/macos.markdown
install/source.markdown
install/windows.markdown

Work around: change the files endings from .markdown to .md

idk if this is intentionally done another reason but i figured it's worth a mention